src/crimson/os/seastore/omap_manager.h

```diff
@@ -151,9 +151,27 @@ class OMapManager {
   virtual omap_list_ret omap_list(
```

Comment:
Add an interface comment that the `first` and `last` must alive during `omap_list()`.